operator/config/crd/bases/forklift.konveyor.io_ovirtvolumepopulators.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
can we make the CRD deployment optional? I think about the case that CDI would also deploy that CRD (I don't think that the flow of deploying forklift and then kubevirt is that likely to happen - it's more likely that kubevirt will be there already when forklift is deployed - would the operator just "skip" that or fail?)

Reply:
i don't think so. @mnecas am i right?

Reply:
This is our custom CRD, so this is fine.
Plus, if we would use some other CRD, like from the volume-data-source-validator. We should define it in our `owned` CRDs [1]. Multiple operators can own the CRD. Once some operator wants to change/update the CRD it will not let them until all operators allow it [2].

---

Dependency resolution of multiple operators [2]
```
OLM will upgrade CRD right away if it is owned by singular CSV. If CRD is owned by multiple CSVs, then CRD is upgraded when it is satisfied all of the following backward compatible conditions...
```

The design documentation philosophy [3].
```
An operator should declare the CRDs it owns and manages, as well as those that it expects to exist (but be managed by other operators).
```

[1] https://github.com/kubev2v/forklift/blob/main/operator/config/manifests/bases/forklift-operator.clusterserviceversion.yaml#L34-L75
[2] https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/dependency-resolution.md
[3] https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/philosophy.md

Reply:
the v olume-data-source-validator dropped. it works without it, although it's a good thing to verify the source ref. if a user wish to have it - he can deploy it.
as for the OvirtVolumePopulator - then i think it's fine.

Reply:
> This is our custom CRD, so this is fine. 

Yes, but the context is that we'd like to let CDI deploy it so it would be available on remote clusters